src/test/java/edu/rosehulman/quota/GetUserControllerTest.java

```diff
@@ -0,0 +1,55 @@
+package edu.rosehulman.quota;
+
+import edu.rosehulman.quota.controller.GetUserController;
+import edu.rosehulman.quota.model.User;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import spark.Request;
+import spark.Response;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Database.class, Request.class, Response.class, User.class})
+public class GetUserControllerTest {
+
+  @Test
+  public void testGetUser() throws Exception {
+    mockStatic(Database.class);
+    Database database = mock(Database.class);
+    when(Database.getInstance()).thenReturn(database);
+
+    mockStatic(User.class);
```

Comment:
I'm pretty sure you don't need this line. We are not mocking static methods in user.